app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -43,16 +43,29 @@
       </div>
 
       <SectionHeader title="Description" />
-
       <div class="border-b border-grey-100">
         <p style="white-space: pre-line" class="intent px-4 md:px-12 py-12 mx-auto max-w-6xl">
           {{ grantMetadata?.description }}
         </p>
       </div>
 
       <!-- LINKS -->
-      <SectionHeader title="Links" />
-      <div class="px-4 md:px-12 py-8 border-b border-grey-100 flex flex-col gap-y-4">
+      <SectionHeader
+        v-if="
+          isDefined(grantMetadata?.properties?.websiteURI) ||
+          isDefined(grantMetadata?.properties?.githubURI) ||
+          isDefined(grantMetadata?.properties?.twitterURI)
+        "
+        title="Links"
+      />
+      <div
+        v-if="
+          isDefined(grantMetadata?.properties?.websiteURI) ||
+          isDefined(grantMetadata?.properties?.githubURI) ||
+          isDefined(grantMetadata?.properties?.twitterURI)
```

Comment:
I think it'd be cleaner and more readable to pull these `isDefined` checks into a computed property instead of redefining it twice in the template

Reply:
Similarly, could wrap the whole LINKS section in one div that way you only need one `v-if` statement

Reply:
i'll fix this up